M2Mqtt/Messages/MqttMsgPubcomp.cs

```diff
@@ -24,20 +27,30 @@ namespace nanoFramework.M2Mqtt.Messages
     /// </summary>
     public class MqttMsgPubcomp : MqttMsgBase
     {
+        /// <summary>
+        /// Return Code, v5.0 only
+        /// </summary>
+        public MqttReasonCode ReasonCode { get; set; }
+
+        /// <summary>
+        /// The REason as a string, v5.0 only
```

Comment:
Typo in reason (there are other, so better go with a find & replace 😉)

Reply:
yes, good catch :-)